timrlink.net.CLI/Actions/ProjectTimeImportAction.cs

```diff
@@ -40,13 +42,25 @@ public sealed override async System.Threading.Tasks.Task Execute()
 
         private async System.Threading.Tasks.Task ImportProjectTimeRecords(IList<Core.API.ProjectTime> records)
         {
+            var users = await UserService.GetUsers();
+            var userDictionary = users
+                .Where(user => user.externalId != null)
+                .ToDictionary(user => user.externalId);
             var tasks = await TaskService.GetTaskHierarchy();
+
             var taskDictionary = await TaskService.CreateExternalIdDictionary(tasks,
                 task => task.parentExternalId != null ? task.parentExternalId + "|" + task.name : task.name
             );
-
+            
             foreach (var record in records)
             {
+                var externalUserId = record.externalUserId;
+                if (!userDictionary.ContainsKey(externalUserId))
+                {
+                    Logger.LogError($"User with ExternalId {externalUserId} not found. Skipping ProjectTime to import.");
```

Comment:
would be good if we would log the skipped record here, otherwise the skipped entry doesn't show up anywhere